tools/interop_matrix/create_matrix_images.py

```diff
@@ -77,6 +77,13 @@
                   action='store_true',
                   help='keep the created local images after uploading to GCR')
 
+argp.add_argument('--reuse_git_root',
```

Comment:
IMHO it's not very clear what the option does.  Can you either rename the option on improve the helptext.
My understanding is it that if False, it does it wipes out git rood directory after each build and does a clean checkout.

Reply:
Done.